cmd/alertmanager/main.go

```diff
@@ -119,11 +125,18 @@ func main() {
 		routePrefix   = kingpin.Flag("web.route-prefix", "Prefix for the internal routes of web endpoints. Defaults to path of --web.external-url.").String()
 		listenAddress = kingpin.Flag("web.listen-address", "Address to listen on for the web interface and API.").Default(":9093").String()
 
-		meshListen = kingpin.Flag("mesh.listen-address", "mesh listen address. Pass an empty string to disable.").Default(net.JoinHostPort("0.0.0.0", strconv.Itoa(mesh.Port))).String()
-		hwaddr     = kingpin.Flag("mesh.peer-id", "mesh peer ID").Default(mustHardwareAddr()).String()
-		nickname   = kingpin.Flag("mesh.nickname", "mesh peer nickname").Default(mustHostname()).String()
-		password   = kingpin.Flag("mesh.password", "password to join the peer network (empty password disables encryption)").Default("").String()
-		peers      = kingpin.Flag("mesh.peer", "initial peers (may be repeated)").Strings()
+		clusterBindAddr = kingpin.Flag("cluster.address", "listen address for cluster").
```

Comment:
cluster.listen.address would be clearer, this current reads as the cluster to talk to